db/migrate/20220709112249_add_user_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] Kindly use the correct indexes given in the ERD 'author_id'

Reply:
Hi i referenced user_id because its the user table that's created. Which has been updated.